aerospike_helpers/expressions/list.py

```diff
@@ -358,13 +358,14 @@ class ListRemoveByValue(_BaseExpr):
 
     _op = aerospike.OP_LIST_REMOVE_BY_VALUE
 
-    def __init__(self, ctx: "TypeCTX", value: "TypeValue", bin: "TypeBinName"):
+    def __init__(self, ctx: "TypeCTX", value: "TypeValue", bin: "TypeBinName", inverted: bool = False):
         """Args:
             ctx (TypeCTX): An optional list of nested CDT :mod:`cdt_ctx <aerospike_helpers.cdt_ctx>` context operation
                 objects.
             value (TypeValue): Value or value expression to remove.
             bin (TypeBinName): bin expression, such as :class:`~aerospike_helpers.expressions.base.MapBin` or
                 :class:`~aerospike_helpers.expressions.base.ListBin`.
+            inverted (bool): Return the opposite expected results of list command.
```

Comment:
nit: I find this doc string confusing, maybe something closer to https://docs.aerospike.com/server/guide/data-types/cdt-list-ops#return-types like "Invert the expression's search criteria"?

Reply:
Fixed